### ISSUE TITLE
Upgrading Geolocator

### ISSUE THREAD

**Issue by faroukcharkas** [medyas/flutter_qiblah#32]
This is a beautiful package and very helpful! I was hoping you could update the Geolocator dependency to the most recent version as being forced to use an older version of geolocator is breaking my app?

JzK!

**Comment by omar-danasoury** [comment]
Hello, I want to contribute to this project. plz assign this task to me. Kind regards

**Comment by medyas** [comment]
@omar-danasoury your welcome to creating a pull request

**Comment by ahmedalmaz** [comment]
@medyas  please can u finish it and ugrade geolocator kindly

**Comment by omar-danasoury** [comment]
hello, sorry for the late reply.
I already started working on the issue but I found that the app needs to upgrade the android embedding from v1 to v2. I faced this issue on an android device and I am not sure if this problem exists on iOS devices. I tried to solve it but unfortunately, there is much code that I do not know, and as a first-timer in this project, I think this issue needs someone who is well experienced with the project. Therefore, I apologize for not doing this task and I hope to contribute to other tasks inshallah in the future.
Kind regards,
